ynput/ayon-backend
# Addon compatibility check for bundles

https://github.com/ynput/ayon-backend/issues/123

## Testing
- Create an addon with dependencies (as described in the linked issue)
- Try creating a bundle that does not meet the dependencies
- It should display an error

When the frontend counterpart of this PR is completed, the frontend will display errors and warnings inline.
